build.gradle

```diff
@@ -395,8 +399,7 @@ bintray {
 }
 
 release {
-  dependsOn distributionPackage
-  push = true
+  push = false
```

Comment:
How does this work if executed on the build server? Then the change would only be locally there?